pkg/cluster/k8sres_test.go

```diff
@@ -1865,7 +1884,7 @@ func TestAdditionalVolume(t *testing.T) {
 		{
 			subTest:        "checking volume mounts of postgres container",
 			container:      constants.PostgresContainerName,
-			expectedMounts: []string{"pgdata", "test1", "test3", "test4"},
+			expectedMounts: []string{"pgdata", "test1", "test3", "test4", "test5", "test6"},
```

Comment:
Please extend this unit test to by adding `expectedSubPath` and `expectedSubPathExpr` to the test struct so you can compare it later when looping over the test cases.